### ISSUE TITLE
Add labels to markers

### ISSUE THREAD

**Issue by jgarber79** [jupyter-widgets/ipyleaflet#804]
Is it possible to add labels to markers? I'm thinking something along the lines of placemark names in GE/KML. I'm aware of the title attribute for markers, but that doesn't seem like the intended purpose. I'd like the labels to be visible always (or not, set by attribute) with some level of control on their appearance (size, color). I'd like to be able to display a friendly name or numerical id for each marker. Is this possible, and if not, are there workarounds? Anyone implemented this on their own fork?

**Comment by giswqs** [comment]
Yes, this would be a nice feature to add. Also see https://github.com/jupyter-widgets/ipyleaflet/issues/792
